src/ic-xrc-types/src/lib.rs

```diff
@@ -19,13 +19,24 @@ pub struct Asset {
 }
 
 /// The type the user sends when requesting a rate.
+///
+/// Suppose BTC/USDT is 30_000. This means that you have to give 30_000 USDT in
+/// order to receive 1 BTC. For people with a non-economics background, this is
+/// completely backwards: you would say that the "rate" in this example is
+/// "30_000 USDT per BTC", which would be written as "30_000 USDT/BTC".
+/// Economists do NOT read "BTC/USD" as "BTC per USD". In this example, BTC is
+/// the "base" asset, and USDT is the "quote" asset.
+///
+/// Translation table for non-economists:
+///
+///             X/Y = Ys per X
+///      base asset = resource that you want to receive
+///     quote asset = resource that you are offerring
```

Comment:
I like the intent but it's too verbose for my liking.
How about we just refer to the definition of [currency pair](https://en.wikipedia.org/wiki/Currency_pair)?
I think it's more important to point out that `X` is the base asset and `Y` is the quote asset in the expression `X/Y` and explain "base asset" and "quote asset". Of course, adding a small example may help.